homeassistant/components/tahoma/cover.py

```diff
@@ -137,14 +137,13 @@ def update(self):
         if self._closure is not None:
```

Comment:
The comment above this line is no longer correct.

Reply:
Thanks I'll update the comment.

Reply:
It seems to be valid for me.
For non horizontal awning, `self._position = 100 - self._closure`, ie: when the roller shutter is closed (`= 100`), the position is equal to `0`.
And then the closed state is `true` (`self._position == 0`).

Reply:
The comment states that `'core:ClosureState': 100 is closed, 0 is fully open.` yet below we check `self._closed = self._position == 0`

Reply:
Yes @balloob this is valid for me:
- `core:ClosureState` corresponds to the value `self._closure`.
- ` #   _position: 0 is closed, 100 is fully open.`.
- `#   'core:ClosureState': 100 is closed, 0 is fully open.`.
- for non `HORIZONTAL_AWNING` device : `self._position = 100 - self._closure`.
- So when `self._closure == 100` (closed state), `self._position = 0`, right?
- And when `self._closure == 0` (opened state), `self._position = 100`, right?
- Then `self._closed = self._position == 0`:
  - when `self._closure == 100`, `self._position = 0`, `self._closed = true`.
  - when `self._closure == 0`, `self._position = 100`, `self._closed = false`.

So for me the comment seems to be valid, and shows that the code may be unnecessarily complicated 😓

Reply:
Yeah, that's definitely complicated. This should be cleaned up (in a new PR).